Documentation/Templating/Index.rst

```diff
@@ -0,0 +1,72 @@
+.. include:: ../Includes.txt
+
+
+.. _Templating:
+
+==========
+Templating
+==========
+
+At this point, you should now be able to install and configure TYPO3 and be familiar with
+its backend. You may be wondering how you go about adding bespoke HTML templates
+to your own TYPO3 project and you might have noticed that the HTML, CSS and Javascript for the Introduction Package are
+are not accessible in the backend.
+
+Whilst this tutorial does not cover templating in TYPO3, this section is intended to answer both of these questions
+and give you a brief introduction to TYPO3's templating engine and also discuss best practices for managing your sites
+HTML templates, stylesheets and other assets.
+
+.. _Introducing Fluid:
```

Comment:
I'm not 100% sure, but I guess there should be no spaces within the target.

I would also propose to lowercase all words within the targets.